tedana/decomposition/_utils.py

```diff
@@ -31,7 +37,10 @@ def eimask(dd, ees=None):
         ees = range(dd.shape[1])
     imask = np.zeros((dd.shape[0], len(ees)), dtype=bool)
     for ee in ees:
```

Comment:
What do you think about checking the size of `ees`. If `len(ees) < dd.shape[1]`, then it's either optcom or a subset of echoes. If `len(ees) == 1`, it's optcom, and if it's greater than 1 but less than the shape of the data, then it's a subset.

Reply:
Should we differentiate between three categories in logging, that is: Optcom, all echoes, or subset? I can see the logging message getting a little confusing - in that echo number could refer to the actual eche number or just indicate how many echoes it performed the function on. 

Maybe that is accounted for. This is my first real pull request so I'm still sorting things out.

Reply:
It is not accounted for yet. Perhaps a good simplified solution would be:

```python
if dd.shape[1] == 1:
    LGR.debug('Creating eimask for optimal combination')
else:
    LGR.debug('Creating eimask for echo {}'.format(ee))
```

This way we can keep the inputs to this function simple (i.e., not include `ste`).